### PR TITLE
Created the notes listing page UI

### LINKED ISSUE #1: Create note listing page UI

### PR DESCRIPTION
**Pull request by allenjiji**

### CODE REVIEW

File: app/javascript/src/components/Common/Sidebar/index.jsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+// @ts-nocheck
```
Comment (yedhink) [inline app/javascript/src/components/Common/Sidebar/index.jsx:1]: I want you go through the Slack thread where I have added review comments and fix the issue over here.

File: app/javascript/src/components/Dashboard/Notes/Card/Card.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
+// @ts-nocheck
+import React from "react";
+
+import { MenuVertical, Clock } from "@bigbinary/neeto-icons";
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/Card/Card.jsx:4]: It seems to me that you haven't gone through the Slack thread very well. Go through it, fix the whole PR and then add a comment and re-request my review once done.

File: app/javascript/src/components/Dashboard/Notes/Card/Card.jsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+import { Header } from "neetoui/layouts";
+
+import { formatDateTime, getRelativeTime } from "./utils";
+
+const dropdown = (
+  <Dropdown buttonStyle="text" icon={MenuVertical}>
+    <li> Edit </li>
+    <li>Delete</li>
+  </Dropdown>
+);
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/Card/Card.jsx:15]: It seems to me you haven't gone through the React best practices book very well. Go through it very well and make necessary changes in this PR. I don't want to pinpoint sections from the book again like I am doing right now:
https://academy.bigbinary.com/learn-react/pages/naming-conventions-in-react#:~:text=When%20to%20use%20the%20render%20keyword%3F

**Review by yedhink — COMMENTED** [review]
@allenjiji The title of this PR doesn't align with the guidelines mentioned at https://handbook.neetokb.com/articles/git-and-github-essentials. Make sure to go through it and also the following comments.

File: app/javascript/src/components/Common/Sidebar/index.jsx (hunk 1 above)
Comment (allenjiji) [reply app/javascript/src/components/Common/Sidebar/index.jsx:1]: I will definitely go through them right now and will do the needful to fix the issue.

File: app/javascript/src/components/Dashboard/Notes/Card/Card.jsx (hunk 2 above)
Comment (allenjiji) [reply app/javascript/src/components/Dashboard/Notes/Card/Card.jsx:4]: I have gone through the slack thread and identified not using the aliases in my code.
I have corrected the known mistakes and I am sending another PR along with the Notes deletion UI.
Is that fine?

**Comment by allenjiji** [comment]
@yedhink _a 
As you [said](https://github.com/allenjiji/neeto-ui-challenge-allenjiji/pull/7#discussion_r973065058), I've gone through the slack thread and React Best Practices once again.
I believe all the mistakes made are taken care of and the next [PR](https://github.com/allenjiji/neeto-ui-challenge-allenjiji/pull/8) is sent.
Requesting a review again.

**Pull request merged**